client/asset/dcr/dcr.go

```diff
@@ -825,6 +847,11 @@ func unconnectedWallet(cfg *asset.WalletConfig, dcrCfg *walletConfig, chainParam
 		return nil, err
 	}
 
+	dir := filepath.Join(cfg.DataDir, chainParams.Name)
+	if err := os.MkdirAll(dir, 0755); err != nil {
```

Comment:
Although we use `0755` elsewhere in the repo, I'm thinking we should set this to `0744`. Only users should r,w,x others should only r.

Reply:
`x` is the permission bit to list a folders contents. `r` alone does not do that.  If you deny execute for a folder, you might as well deny read as well.

Reply:
Leaving I guess? I don't know if other values have some unintended effects.

I think I added this so there is a place for for the vsp file, but I think it is made at some point after with the 0755 permissions if not here....

Reply:
> Leaving I guess? 

Yes, we can keep `0755`. @chappjc agrees it's better.

We also use `0744` elsewhere, but maybe it does not apply.

Reply:
> We also use `0744` elsewhere, but maybe it does not apply.

That sounds like an error and/or copy pasta that has spread.  There are indeed many such lines, and git blame says that myself and others have done this, hence it smells like copy pasta.

Anyone else have an opinion about this?  0744 makes little sense to me.

Reply:
755